tools/generator/templates/player/python/Makefile.jinja2

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+{% import 'macros/make.jinja2' as make %}
+
+# Add all your sources (must be suffixed by *.py) in here, separated by spaces
```

Comment:
*(must end with .py)* sounds simpler

Reply:
Will fix when refactoring all the makefiles